search-app/resources/dependencycheck.properties

```diff
@@ -0,0 +1,4 @@
+analyzer.ruby.gemspec.enabled=false
```

Comment:
Can we comment on whats going on with these properties, why they are needed etc.